### ISSUE TITLE
setup metabase again / create new admin account

### ISSUE THREAD

**Issue by nha** [metabase/metabase#2826]
I would like to delete the stored data as it appears that I used metabase a long time ago and lost my credentials. My emails apparently are not stored (so I cannot recover the passwords). And I cannot create a new admin account.

*  I am using the chrome browser.
*  My computer's OS is OSX el Capitan.
*  I'm running Metabase locally with the Mac App.
*  My database is mongoDB.
*  My Metabase version is 0.18.0.2.

**Comment by salsakran** [comment]
The embedded application database file is stored at
`~/Library/Application\ Support/Metabase/metabase.db.h2.db `

If you delete this, all information from your previous account will be deleted. Alternatively, you can reset the password via the `Reset Password` link in the Help menu.

**Comment by nha** [comment]
Removing that file worked. The onboarding is nice !

(I could not reset my email because it was not existing apparently - but I tried metabase a long time ago and could not remember which email I used).

Thanks !

**Comment by sventech** [comment]
FYI, on my system the file was
`~/Library/Application\ Support/Metabase/metabase.db.mv.db`

**Comment by faisaliftikhar66** [comment]
Hi guys , what is the path for windows 10 ?

**Comment by nimrod-cohen** [comment]
Similar question, found metabase.db.mv.db on my root folder (centos 7), moved it, but the app keeps on running as if nothing changed.
Also couldn't find any metabase.db.h2.db file anywhere.

I'm the admin, I'm not sure the email smtp details have been configured, trying to log in, I'm only getting contact your administrator. 
I'm in a loop here.

**Comment by nimrod-cohen** [comment]
also cannot get to the help menu, because i'm not logged in, so I'm completely locked out.
Also tried to delete the metabase.jar file on the root directory of the website. nothing budged.

**Comment by NikolaZavadska** [comment]
Hi :), did someone managed to solve this? I deleted everything that was pointing to Metabase, nothing is in Application support. I Installed Metabase again and it still shows login page not setup page.

**Comment by Sven-vinna** [comment]
I was able to solve this in Debian Stretch after renaming two db files on the same directory where I have my metabase.jar file. The files were `metabase.db.mv.db` and  `metabase.db.trace.db`, after renaming them I was able to start at the setup again.

**Comment by plaufont** [comment]
I well want an answer to this question.
I am running metabase on Windos Server 2012. I am trying to restart from scrach the metabase configuration, after having deleted the .db files, the plugins, and everything i could find, Metabase still ask me to login when I connect localhost:3000

**Comment by paperguns** [comment]
I had this issue as I forgot the password to the development branch I was working on. @sventech 's solution is the right direction to hard reboot your dev instance which will reset all instance changes (be careful if you have important work like dashboards or questions). I had to stop and start my dev instance for the change to have an impact. Once I restarted the branch the visit to localhost:3000 loading Metabase at the configuration step and I was able to set up a new admin password.
